modules/tinymce/src/themes/silver/main/ts/ui/core/ChoiceControls.ts

```diff
@@ -132,7 +137,14 @@ const languageSpec = (editor: Editor): Optional<ControlSpec<ContentLanguage>> =>
           })
       );
     },
-    setCurrent: (editor, lang) => editor.execCommand('Lang', false, lang)
+    setCurrent: (editor, lang) => editor.execCommand('Lang', false, lang),
+
+    onToolbarSetup: (api) => {
+      const unbinder = Singleton.unbindable();
+      api.setActive(editor.formatter.match('lang', {}, undefined, true));
```

Comment:
Shouldn't it be an `optional` rather than `undefined`?

Reply:
No, it's a public API so it can't be using `Optional`.